deme/tests/expand-demes-testold.metta

```diff
@@ -112,4 +123,4 @@
 ; (>= $topCandidatesLength 1)
 ; ) True)
 
-! (runMoses 1 (mkCscore -0.1 2 0.1 0.1 3) 3 metaPop2 0 2 table hillClimbing 5 2 0 1 ttable1 3 0.004 1000)
+;! (runMoses 1 (mkCscore -0.1 2 0.1 0.1 3) 3 metaPop2 0 1 table hillClimbing 5 2 0 1 ttable1 3 0.004 1000)
```

Comment:
Okay why did you comment this code out? Is it causing problems? And why change the number of demes to 1?